### PR TITLE
fixed documentation spelling

### PR DESCRIPTION
**Pull request by Meir017**

### CODE REVIEW

**Comment by Meir017** [comment]
I noticed that most of the files in the directory `Operations` have the using `using System.Collections.Immutable;` for no reason

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/analyzer-ioperation

**Comment by CyrusNajmabadi** [comment]
@jaredpar As this is just doucmentation comment fixes, can we eschew the double compiler review?

File: src/Compilers/Core/Portable/Operations/IConditionalAccessExpression.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public interface IConditionalAccessExpression : IOperation
         /// </summary>
         IOperation ConditionalValue { get; }
         /// <summary>
-        /// Expresson that is conditionally accessed.
+        /// Expression that is conditionally accessed.
```
Comment (333fred) [inline src/Compilers/Core/Portable/Operations/IConditionalAccessExpression.cs]: Fixed by https://github.com/dotnet/roslyn/pull/21624, please revert to not conflict.

Comment (Meir017) [reply src/Compilers/Core/Portable/Operations/IConditionalAccessExpression.cs]: Reverted

**Comment by Meir017** [comment]
@333fred any update on this?

**Comment by 333fred** [comment]
@dotnet-bot retest windows_release_vs-integration_prtest please

**Comment by 333fred** [comment]
@Meir017 thanks for the contribution! Sorry we took so long in getting it merged.

**Pull request merged**